packages/coinjoin/src/utils/roundUtils.ts

```diff
@@ -158,36 +159,26 @@ const getDataFromRounds = (rounds: Round[]) => {
     };
 };
 
-/**
- * Transform from coordinator format to coinjoinReducer format `CoinjoinClientFeeRatesMedians`
- * array => object { name: value-in-vBytes }
- */
-export const transformFeeRatesMedians = (medians: CoinjoinStatus['coinJoinFeeRateMedians']) => {
-    const [fast, recommended] = medians.map(m => m.medianFeeRate);
-    // convert from kvBytes (kilo virtual bytes) to vBytes (how the value is displayed in UI)
-    const kvB2vB = (v: number) => (v ? Math.round(v / 1000) : 1);
-
-    return {
-        fast: kvB2vB(fast) * 2, // NOTE: this calculation will be smarter once have enough data
-        recommended: kvB2vB(recommended),
-    };
-};
-
 /**
  * Transform from coordinator format to coinjoinReducer format `CoinjoinClientInstance`
  * - coordinatorFeeRate: multiply the amount registered for coinjoin by this value to get the total fee
- * - feeRatesMedians: array => object with values in kvBytes
+ * - maxMiningFee: array => value in kvBytes
  */
 export const transformStatus = ({
     coinJoinFeeRateMedians,
     roundStates: rounds,
 }: CoinjoinStatus) => {
-    const feeRatesMedians = transformFeeRatesMedians(coinJoinFeeRateMedians);
     const { allowedInputAmounts, coordinationFeeRate } = getDataFromRounds(rounds);
+    // coinJoinFeeRateMedians include an array of fast, recommended, and slow medians - we only care about the second (recommended) fee rate
```

Comment:
maybe confusing, it's not fast/recommended/slow medians but medians from different time ranges (day/week/month or so) so it is possible that over time we may find that we should take a different median as the recommended value... 

```suggestion
    // coinJoinFeeRateMedians include an array of medians per day, week and month - we  take the second (week) median as the recommended fee rate
```

Reply:
Done!